kernels/optimized/lib_defs.bzl

```diff
@@ -156,8 +156,11 @@ def define_libs():
                     ],
                 ),
             ],
-            fbobjc_compiler_flags = [
-                "-march=armv8+bf16",
+            fbobjc_platform_compiler_flags = [
+                (
+                    ".*arm64.*",
+                    ["-march=armv8+bf16"],
```

Comment:
I'm not sure how this fix the issue, maybe some comments can help

Reply:
Wouldn't that crash on older phones/laptops?